scripts/Inelastic/CrystalField/fitting.py

```diff
@@ -1289,6 +1292,112 @@ def monte_carlo(self, **kwargs):
             self._monte_carlo_single(**kwargs)
         self.model.FixAllPeaks = fix_all_peaks
 
+    def two_step_fit(self, OverwriteMaxIterations: list = None, OverwriteMinimizers: list = None, Iterations: int = 20) -> None:
+        from mantid.kernel import logger
+        logger.notice("Please note that this is a first experimental version of the two_step_fit algorithm.")
```

Comment:
Can we bump this to a warning?
I'm guessing it may produce invalid data and I think a warning will implore users to actually check their fitting results more carefully